IEXSharp/Helper/ExecutorREST.cs

```diff
@@ -64,21 +66,24 @@ public ExecutorREST(HttpClient client, string publishableToken, string secretTok
 				try
 				{
 					content = await responseContent.Content.ReadAsStringAsync();
-					if (content.Length > 7 && content.Substring(startIndex: 0, length: 8) == "{\"error\"")
-					{ // {"error":"child \"token\" fails because [\"token\" must be a string]"}
-						var token = JToken.Parse(content);
-						return new IEXResponse<ReturnType>() { ErrorMessage = token["error"].ToString() };
-					}
-					else if (content.Equals("forbidden", StringComparison.InvariantCultureIgnoreCase)
-						|| content.Equals("not found", StringComparison.InvariantCultureIgnoreCase))
-					{ // "Forbidden" or "Not found"
-						return new IEXResponse<ReturnType>() { ErrorMessage = content };
+					if (responseContent.StatusCode == HttpStatusCode.OK)
+					{ // Successful Request
+						if (!typeof(IEnumerable).IsAssignableFrom(typeof(ReturnType))
```

Comment:
Toying with the idea of wrapping the return in a try catch and on the (hopefully) rarer catch cases just returning null. 

```csharp
try
{
	return new IEXResponse<ReturnType>()
	{
		Data = JsonConvert.DeserializeObject<ReturnType>(content, jsonSerializerSettings)
	};
}
catch
{
	return null;
}
```

Reply:
This idea is interesting too, but I also think this one is not ideal because it can catch all sorts of exceptions, not just the specific case of getting an empty array when expecting a single value. Thus, it could mask other parsing errors such as if the JSON model becomes out of date. In that case, the user wouldn't get an exception or any indication that something is wrong - they'd just get a null (which would then get converted to an empty List).